firebase-perf/src/main/java/com/google/firebase/perf/util/Constants.java

```diff
@@ -58,6 +58,9 @@ public class Constants {
   /** Attribute key for the hosting activity of a fragment screen trace. */
   public static final String ACTIVITY_ATTRIBUTE_KEY = "Hosting_activity";
 
+  /** Attribute value for when the current fragment does not have a parent fragment. */
+  public static final String NO_PARENT_FRAGMENT = "No parent";
```

Comment:
nit: rename to FRAGMENT_CUSTOM_ATTRIBUTE_VALUE_NO_PARENT_FRAGMENT?

Reply:
how about PARENT_FRAGMENT_ATTRIBUTE_VALUE_NONE?

Reply:
Makes sense to me!